README.md

```diff
@@ -125,7 +125,7 @@ tw CLI is a platform binary executable created by a native compilation from Java
 2. Install GraalVM:
 
     ```bash
-    sdk env install
+    sdk env install graalvm
```

Comment:
Why did you needed to add this? What error did you got? By default it uses what is defined at `.sdkmanrc` that is `java=17.0.8-graalce`. I've checked to run only `sdk env install` and it still works for me.

Reply:
In fact, after checking the SDKMan docs, I do not find what adding an extra parameter after `env install` does.

Reply:
I've updated the GraalVM install instructions to explicitly define the correct Java version in `.sdkmanrc`.

Reply:
Not sure that this is needed. When you have a `.sdkmanrc` at the root of your project, it has a preference, so if you go to the `tower-cli` location and you do an `sdk env install`, it should use only this `.sdkmanrc` and ignore any other `.sdkmanrc`. 

So maybe what is missing in the `README` is to make explicit that you need to be at the root of `tower-cli` folder before doing the `sdk env install`.